sources/web/datalab/static/appbar.html

```diff
@@ -18,42 +18,42 @@
       </button>
     </div>
     <div class="btn-group">
+      <a href="/sessions" target="_blank" id="sessionsButton">
+        <button title="Running Sessions" class="toolbar-btn">
+          <i class="material-icons">playlist_play</i>
+        </button>
+      </a>
+    </div>
+    <div class="btn-group">
       <button class="toolbar-btn" data-toggle="dropdown" title="Help Links">
-        <span class="fa fa-question-circle"></span>
+        <i class="material-icons">open_in_new</i>
       </button>
       <ul class="dropdown-menu dropdown-menu-right" style="width: 200px">
         <li><a id="keyboardHelpLink" style="display:none" href="#">Keyboard Shortcuts</a></li>
         <li><a id="markdownHelpLink" style="display:none" href="#">Markdown Syntax</a></li>
         <li id="notebookHelpDivider" class="divider" style="display:none"></li>
-        <li><a href="https://cloud.google.com/bigquery/what-is-bigquery" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>BigQuery&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
-        <li><a href="https://cloud.google.com/storage/docs/overview" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>Cloud Storage&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
+        <li><a href="https://cloud.google.com/bigquery/what-is-bigquery" target="_blank"><span>BigQuery&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
+        <li><a href="https://cloud.google.com/storage/docs/overview" target="_blank"><span>Cloud Storage&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></a></li>
         <li class="divider"></li>
-        <li><a href="https://developers.google.com/chart/" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>Google Charts</span></a></li>
-        <li><a href="http://matplotlib.org/contents.html" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>Matplotlib</span></a></li>
-        <li><a href="http://pandas.pydata.org/pandas-docs/stable/" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>Pandas</span></a></li>
-        <li><a href="http://scikit-learn.org/stable/" target="_blank"><i class="fa fa-external-link pull-right menu-icon"></i><span>SciKit-Learn</span></a></li>
+        <li><a href="https://developers.google.com/chart/" target="_blank"><span>Google Charts</span></a></li>
+        <li><a href="http://matplotlib.org/contents.html" target="_blank"><span>Matplotlib</span></a></li>
+        <li><a href="http://pandas.pydata.org/pandas-docs/stable/" target="_blank"><span>Pandas</span></a></li>
```

Comment:
I realize I am trying to squeeze in something extra here -- if you're ok:

Add Numpy before pandas - https://docs.scipy.org/doc/numpy/reference/
Add TensorFlow after SciKit-Learn - https://tensorflow.org